frontend/src/pages/auth/signup.tsx

```diff
@@ -1,11 +1,6 @@
-/* eslint-disable @typescript-eslint/no-misused-promises */
-import { SingInLayout } from "@components/_shared/SignInLayout";
 import type { GetServerSideProps } from "next";
-import { getCsrfToken, signIn } from "next-auth/react";
-import { NextSeo } from "next-seo";
-import { useRouter } from "next/router";
-import { deleteCookie, setCookie } from "cookies-next";
-import { useEffect } from "react";
+import { getCsrfToken } from "next-auth/react";
+import LoginPage from "@components/auth/LoginPage";
 
 export const getServerSideProps: GetServerSideProps = async (context) => {
```

Comment:
You can remove this function as you created a wrapper for the entire page.

Reply:
I think it might be somehow necessary